backend/api/Areas/Admin/Models/User/AccessRequestUserModel.cs

```diff
@@ -43,6 +43,12 @@ public class AccessRequestUserModel : Pims.Api.Models.BaseModel
         /// </summary>
         /// <value></value>
         public string Email { get; set; }
+        
+        /// <summary>
+        /// get/set - The username.
+        /// </summary>
+        /// <value></value>
+        public string UserName { get; set; }
```

Comment:
This comes down to preference but what I see all around the web is "username" in JSON, which would map to "Username" here. That would also simplify the mapping above where the Entity field is named "Username" too